src/components/search/character-check.js

```diff
@@ -24,43 +26,46 @@ export default class CharCheck {
   updateCheckReadout(event, firstRun) {
     const value = this.input.value;
     const remaining = this.checkVal - value.length;
+    const noMaxCheck = this.checkMinOnly;
     // Prevent aria live announcement when component initialises
     if (!firstRun && event.inputType) {
       this.checkElement.setAttribute('aria-live', 'polite');
     } else {
       this.checkElement.removeAttribute('aria-live');
     }
 
-    this.checkRemaining(remaining);
+    this.checkRemaining(remaining, noMaxCheck);
     this.setCheckClass(remaining, this.input, inputClassLimitReached);
     this.setCheckClass(remaining, this.checkElement, remainingClassLimitReached);
   }
 
-  checkRemaining(remaining) {
+  checkRemaining(remaining, noMaxCheck) {
     let message;
     if (remaining === 1) {
       message = this.singularMessage;
-    } else if (remaining === -1) {
+    } else if (remaining > 1) {
+      message = this.pluralMessage;
+    } else if (remaining === -1 && !noMaxCheck) {
       message = this.charLimitSingularMessage;
       remaining = Math.abs(remaining);
-    } else if (remaining < -1) {
+    } else if (remaining < -1 && !noMaxCheck) {
       message = this.charLimitPluralMessage;
       remaining = Math.abs(remaining);
-    } else {
-      message = this.pluralMessage;
     }
-    this.setShowMessage(remaining);
-    this.setButtonState(remaining);
+    this.setShowMessage(remaining, noMaxCheck);
```

Comment:
We don't need to pass it to the function. We can just check `this.checkMinOnly` at the point we need to as it doesn't change after the class is instantiated.